comp/venue/organise-screens.yaml

```diff
@@ -0,0 +1,39 @@
+summary: Organise display screens for the $SRYYYY competition
+
+priority: should
+
+component: Arena
+
+milestone: $SRYYYY Competition
+
+area-owner: event-logistics
+
+description: >-
+  SR owns 6 matching screens, without stands, which are used for the
+  arena displays. These have the required cables to plug into HDMI sources.
+
+  We also have some 19" screens with stands and a variety of other input types.
+
+  We need to figure out what screens we need for the competition and
+  whether we have enough of the right type.
+
+  For the past few years we have used:
+   - 5 screens at the arena (4 corners & staging).
+   - 2/3 screens for the livestream.
+   - 1 screen for scoring entry and comp ops.
+   - Installed screens at the venue presenting at least 1 outside screen and 1 livestream per area of the venue.
+
+  For each screen we will need something to drive it.
+  For the livestream and scoring entry screens these are people's laptops.
```

Comment:
I don't think these interactive workstations should be included in this ticket about the display screens. These workstations form very separate roles to the other displays, have quite different requirements and could easily be organised by someone else.

Reply:
These are listed here because if they are not a laptop they would require a display. The intention here is to make this the single ticket for organising any displays we want for the competition.